includes/admin/views/html-admin-settings.php

```diff
@@ -38,7 +38,7 @@
 			self::show_messages();
 
 			do_action( 'woocommerce_settings_' . $current_tab );
-			do_action( 'woocommerce_settings_tabs_' . $current_tab ); // @deprecated hook. @todo remove in 4.0.
+			do_action( 'woocommerce_settings_tabs_' . $current_tab ); // @deprecated 3.4.0 hook. @todo remove in 4.6.0.
```

Comment:
this should be removed in 4.7 as per plan?

Reply:
@vedanshujain For this one, I should have removed the @todo. Anything 3.0 or later won't be removed this year.